redisinsight/api/src/modules/rdi/client/rdi.client.ts

```diff
@@ -23,6 +23,12 @@ export abstract class RdiClient {
 
   abstract getPipeline(): Promise<RdiPipeline>;
 
+  // TODO validate options and response
```

Comment:
Is this going to be completed in a separate card?

Reply:
Yep, i guess in integration card (i'm asked rdi about request and response, but didn't get answer)